crates/oxc_linter/src/snapshots/max_lines.snap

```diff
@@ -0,0 +1,237 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: max_lines
+---
+  ⚠ eslint(max-lines): "File has too many lines (3). Maximum allowed is 2."
+   ╭─[max_lines.tsx:1:1]
+ 1 │ var xyz;
```

Comment:
Looks like the range of error is not really corresponding :)

Reply:
Thanks for the hint, my usage of Span was not correct! Fixed now.